package.json

```diff
@@ -176,6 +176,19 @@
           "default": "hint",
           "description": "The message severity when a document has no associated grammar.",
           "scope": "window"
+        },
+        "xml.diagnostics.enabled": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enable/disable all diagnostics.",
+          "scope": "window"
+        }
+        ,
+        "xml.diagnostics.schema": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enable/disable schema based diagnostics. Ignored if \"xml.diagnostics.enabled\": false.",
+          "scope": "window"
         }
       }
```

Comment:
I'm thinking we need a consistent namespace for everything, eg. xml.validation.
so:
- xml.validation.enabled
- xml.validation.schema.enabled
- xml.validation.noGrammar
we'd have to deprecate xml.problems.noGrammar